### PR TITLE
[docs] Fix Reference and Example blocks in docs

### PR DESCRIPTION
**Pull request by mpariente**
It annoyed me.. 

Need to add lot of new things in the docs..

### CODE REVIEW

File: asteroid/models/zenodo.py [hunk 1]
```diff
@@ -76,14 +76,14 @@ def change_metadata_in_deposition(self, dep_id, metadata):
...
-            metadata = {
-                'title': 'My first upload',
-                'upload_type': 'poster',
-                'description': 'This is my first upload',
-                'creators': [{'name': 'Doe, John',
-                              'affiliation': 'Zenodo'}]
-            }
+        Examples
+            >>> metadata = {
+            >>>     'title': 'My first upload',
```
Comment (jonashaag) [inline asteroid/models/zenodo.py:81]: all prefixes except the first should be `...`

Comment (mpariente) [reply asteroid/models/zenodo.py:81]: Because it's one instruction?

**Comment by mpariente** [comment]
The docs build doesn't work because of git's deps problems, do you know this problem? Have a solution? @jonashaag

File: asteroid/models/zenodo.py [hunk 2]
```diff
@@ -76,14 +76,14 @@ def change_metadata_in_deposition(self, dep_id, metadata):
...
+            >>>     'upload_type': 'poster',
+            >>>     'description': 'This is my first upload',
+            >>>     'creators': [{'name': 'Doe, John',
+            >>>                   'affiliation': 'Zenodo'}]
+            >>> }
```
Comment (mpariente) [inline asteroid/models/zenodo.py:86]: Like this I guess. 
```suggestion
            >>> metadata = {
            ...     'title': 'My first upload',
            ...     'upload_type': 'poster',
            ...     'description': 'This is my first upload',
            ...     'creators': [{'name': 'Doe, John',
            ...                   'affiliation': 'Zenodo'}]
            ... }
```

File: asteroid/models/zenodo.py (hunk 1 above)
Comment (jonashaag) [reply asteroid/models/zenodo.py:81]: Yep! I assume you wanted it to look like the Python repl.

**Pull request merged**